File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { Magic8BallSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await Magic8BallSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- Magic8Ball SDK exists test

local sdk = require("magic8-ball_sdk")

describe("Magic8BallSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
